### PR TITLE
LPD-13526 new option type

### PR DESCRIPTION
**Pull request by bryceosterhaus**
I added a commit on top of https://github.com/liferay-commerce/liferay-portal/pull/4814

All new JS code should be in a JS file and not in an aui:script tag

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
cool, I'll send this to Brian, thank you!

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_cp_definition_option_value_rel.jspf [hunk 1]
```diff
@@ -227,124 +227,25 @@ long cpDefinitionOptionValueRelId = cpDefinitionOptionValueRelDisplayContext.get
...
-
-		if (nameInput) {
-			var handleOnNameInput = function () {
-				keyInput.value = slugify.default(nameInput.value);
-			};
-
-			nameInput.addEventListener('input', debounce(handleOnNameInput, 200));
-		}
-	</aui:script>
+	<liferay-frontend:components
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_cp_definition_option_value_rel.jspf:230]: ehy @bryceosterhaus I was testing this PR locally and I found some issues that make us think it's safer for everybody to merge your changes after our PRs.
See:
I found this issue with this change of yours:

Screenshot 2024-04-04 at 10 40 40

this I think should be 
```suggestion
<liferay-frontend:component
```
and even with the suggested change I found this issue:

Screenshot 2024-04-04 at 10 44 00

this is not working and we have no bandwidth to fix this issues too.

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
moreover I found there is some more js in the jsp file:

Screenshot 2024-04-04 at 10 44 28

**Comment by andrea-ale-sbarra** [comment]
see here for fixes https://github.com/liferay-commerce/liferay-portal/pull/4834

**Comment by bryceosterhaus** [comment]
@andrea-ale-sbarra if my changes are causing issues for you, you can just drop my commit. I will fix it in my PR after your changes are merged.

**Pull request closed without merging**